### ISSUE TITLE
PR to dev merged with other contributors

### ISSUE THREAD

**Issue by Shreyas-SAS** [llaske/sugarizer-server#303]
I tried to create a PR to fix typos to the dev branch but it seems that the PR is getting merged with other contributor and shows 2 files changed whereas I have changed only 1 file.  
On master Branch there seems to be no conflict but since dev is 3 commits behind master branch. all the commits are added to my PR. 
Can individual PR's not be sent in this case?
This is the first time it happened with me and I request you to help me solve the issue as I couldn't find a solution on the internet.

![image](https://user-images.githubusercontent.com/96542494/158211118-acc30ac7-3e3d-4fe8-944b-f28b9d92ad87.png)

(Not my commit)
![image](https://user-images.githubusercontent.com/96542494/158211339-76ed3637-cd0f-4a43-92a8-d219b7416476.png)

Sorry for the inconvenience.

**Comment by khareyash05** [comment]
Did uh git pull before commiting?

**Comment by Shreyas-SAS** [comment]
> Did uh git pull before commiting?

Yes. Actually the branch dev is 3 commits behind master. So the problem arose. 

I created the changes in master branch of my forked repo and tried to push it on dev here. This created differences. 

Solution was simple. I deleted the files on my dev branch and then just added the new changed files. This then gave no problem.

I am still thankful for your suggestions and time.
